spec/models/group_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  it 'name should be present' do
+    #  let(:author) { User.new(name: 'Test User', email: 'john@test.com', password: '12345678') }
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻